src/structs.jl

```diff
@@ -450,6 +450,41 @@ end
 """
 $(TYPEDEF)
 
+# Fields
+$(TYPEDFIELDS)
+"""
+Base.@kwdef struct Member
```

Comment:
Do these new structs need to be added to https://github.com/LCSB-BioCore/SBML.jl/blob/64ec00b69130a70cf75775792e76c7db6003b1c3/test/common.jl#L22-L33?

Reply:
good point thx, done now